src/main/java/tools/image/ReliefImageBreaker.java

```diff
@@ -112,25 +111,17 @@ public void createMaps() throws IOException {
   }
 
   /**
-   * Asks user wether to do sea zones only or not
+   * Asks user whether to do sea zones only or not.
    *
-   * @return java.lang.boolean TRUE to do seazones only.
+   * @return True to do seazones only.
    */
   private static boolean doSeaZone() {
-    String ans = "";
     while (true) {
-      ans = JOptionPane.showInputDialog(null, "Only Do Sea Zones? Enter [Y/N]");
-      if (ans == null) {
-        System.out.println("Cannot leave this blank!");
-        System.out.println("Retry");
-      } else {
-        if (ans.equalsIgnoreCase("Y")) {
-          return true;
-        } else if (ans.equalsIgnoreCase("N")) {
-          return false;
-        } else {
-          System.out.println("You must enter Y or N");
-        }
+      final String answer = JOptionPane.showInputDialog(null, "Only Do Sea Zones? Enter [Y/N]");
+      if (answer.equalsIgnoreCase("Y")) {
```

Comment:
(Bumped from previous review)

According to the [Javadocs](https://docs.oracle.com/javase/8/docs/api/javax/swing/JOptionPane.html#showInputDialog-java.awt.Component-java.lang.Object-java.lang.String-int-javax.swing.Icon-java.lang.Object:A-java.lang.Object-), `showInputDialog()` can return `null` if the user cancels the dialog.  Wouldn't it be safer to reverse the equality check here and on line 123 below? For example, `"Y".equalsIgnoreCase(answer)`.